accounts-db/src/accounts_db.rs

```diff
@@ -5404,6 +5404,68 @@ impl AccountsDb {
         )
     }
 
+    /// Load account with `pubkey`. `callback` is used to decide whether to
+    /// store the account into read cache after loading.
+    ///
+    /// Return the account and the slot when the account was last stored.
+    /// Return None for ZeroLamport accounts.
+    pub fn load_account_with(
+        &self,
+        ancestors: &Ancestors,
+        pubkey: &Pubkey,
+        callback: impl Fn(&AccountSharedData) -> bool,
+    ) -> Option<(AccountSharedData, Slot)> {
```

Comment:
The returned slot value is currently not being used. Is there a plan to use it in future/follow-on PR?

Reply:
That is now separated into #1350, and it will be used for the program cache index structure replacement.

Reply:
#1350 is merged.